src/styles.ts

```diff
@@ -40,7 +40,9 @@ export const styles = css`
 		padding-top: var(--paper-input-container_-_padding-top, 8px);
 		padding-bottom: var(--paper-input-container_-_padding-bottom, 8px);
 		position: relative;
-
+		border: var(--cosmoz-input-border);
+		border-radius: var(--cosmoz-input-border-radius);
+		max-height: var(--cosmoz-input-max-height);
```

Comment:
These styles are not necessary, as anything on :host can be set from the outside of the component, simply by writing:

```css
comsoz-input {
  border: 1px solid red;
  border-radius: 10%;
  max-height: 200px;
}
```

Reply:
OK. I'm reverting this.